optaplanner-core/src/main/java/org/optaplanner/core/config/localsearch/decider/acceptor/AcceptorConfig.java

```diff
@@ -240,6 +240,7 @@ public void setStepCountingHillClimbingType(StepCountingHillClimbingType stepCou
     // With methods
     // ************************************************************************
 
+    @Deprecated
```

Comment:
Nitpicking: would be nice to include `@deprecated` javadoc tag informing users about which operation replaces this one.

It should probably also appear in the upgrade recipe, right?

Reply:
Yes, you're right in theory, but in practice no one is using this because Acceptor was the only Config class with with methods, it's a deep class and it wasn't documented (still isn't).
This field was deprecated a long time ago and when the with methods were added recently, the deprecated was forgotten to be added for this one.
So I am going to leave it as is, in this case.